Code/Editor/Core/QtEditorApplication_linux.cpp

```diff
@@ -8,11 +8,21 @@
 
 #include "QtEditorApplication.h"
 
+#ifdef PAL_TRAIT_LINUX_WINDOW_MANAGER_XCB
+#include <AzFramework/Input/Buses/Notifications/RawInputNotificationsBus_xcb.h>
+#endif
+
 namespace Editor
 {
-    bool EditorQtApplication::nativeEventFilter(const QByteArray& , void* , long* )
+    bool EditorQtApplication::nativeEventFilter([[maybe_unused]] const QByteArray& eventType, void* message, long*)
     {
-        // TODO_KDAB_LINUX
+        if (GetIEditor()->IsInGameMode())
+        {
+#ifdef PAL_TRAIT_LINUX_WINDOW_MANAGER_XCB
+            AzFramework::RawInputNotificationBusXcb::Broadcast(&AzFramework::RawInputNotificationsXcb::OnRawInputEvent, static_cast<xcb_generic_event_t *>(message));
```

Comment:
Nitpick: space between `xcb_generic_event_t` and `*`

Reply:
Fixed.